sympy/logic/algorithms/dpll2.py

```diff
@@ -35,16 +35,18 @@ def dpll_satisfiable(expr, all_models=False):
     False
 
     """
-    clauses = conjuncts(to_cnf(expr))
-    if False in clauses:
+    if not isinstance(expr, EncodedCNF):
+        exprs = EncodedCNF()
+        exprs.add_prop(expr)
+        expr = exprs
+
+    # DONE: Check to handle this
+    if {0} in expr.data:
```

Comment:
What does this mean?

Reply:
During the encoding, `False` is encoded as `0`. It is used here to test the presence of `False` in `EncodedCNF`.

Reply:
A more meaningful comment might help

Reply:
Ohh, that was a TODO I left. I am changing it.